server/data/lostBedClient.test.ts

```diff
@@ -134,4 +135,36 @@ describeClient('LostBedClient', provider => {
       expect(result).toEqual(lostBed)
     })
   })
+
+  describe('cancel', () => {
+    it('cancels a lost bed', async () => {
+      const lostBed = lostBedFactory.build()
+      const notes = 'note'
+
+      const lostBedCancellationData: NewLostBedCancellation = {
+        notes,
+      }
+
+      provider.addInteraction({
+        state: 'Server is healthy',
+        uponReceiving: 'A request to update a lost bed',
+        withRequest: {
+          method: 'POST',
+          path: paths.premises.lostBeds.cancel({ premisesId, id: lostBed.id }),
+          body: lostBedCancellationData,
+          headers: {
+            authorization: `Bearer ${token}`,
+          },
+        },
+        willRespondWith: {
+          status: 200,
+          body: lostBed,
```

Comment:
should it respond with a LostBed, or a LostBedCancellation?

Reply:
good spot this

Reply:
or bad error on my part 🤷